lib/OpenQA/WebAPI/Controller/Main.pm

```diff
@@ -140,8 +140,17 @@ sub group_overview {
     return $self->reply->not_found unless $group;
 
     my $res = $self->_group_result($group, $limit_builds);
-    my @comments = $group->comments->all;
-    for my $comment (@comments) {
+    my @comments;
+    my @pinned_comments;
+    for my $comment ($group->comments->all) {
+        # find pinned comments
+        if ($comment->user->is_operator && CORE::index($comment->text, 'pinned-description') >= 0) {
```

Comment:
If I'd improve the feature to use a database flag, I wouldn't use a pattern at all.